frontend/src/metabase/entities/actions/actions.ts

```diff
@@ -89,18 +89,6 @@ const Actions = createEntity({
       return card;
     },
     update: updateAction,
-    list: async (props: any) => {
-      const { modelId } = props;
-
-      if (modelId) {
-        const modelActions = await ModelActionsApi.getModelActions({
-          id: modelId,
-        });
-        return modelActions.map(mapModelActionsToActions);
-      }
-
-      return ActionsApi.list(props);
-    },
```

Comment:
the backend abstracted away most of the model-actions v actions distinction, so we don't have to worry about it much on the front end anymore